DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: admiralext
```

Comment:
i think we just call it `admiraltemplate` here and in other parts of the documenation

Reply:
Think I covered this with the most recent commit